### PR TITLE
Use the reference manifest

### PR DESCRIPTION
**Pull request by jhford**
Instead of loading the old apis.json file, we should use the global manifest.json file that other clients use to figure out which components to set up.

The add command is no longer relevant when we're using a manifest.json file

### CODE REVIEW

File: bin/update-apis.js [hunk 1]
```diff
@@ -145,77 +145,42 @@ program
...
+      }));
+    });
+
+    p = p.then(function() {
       updateDocs();
       saveApis();
       browserifyModule();
     });
+
+    p.done();
```
Comment (jonasfj) [inline bin/update-apis.js]: there is no need to do done(), where I would rather see `.catch()` something like this:
```js
.catch(function(err) {
  console.log("Error adding reference: " + err);
});
```

But I guess `done()` works too... it just throws the exception and crashes the application, so it's very rarely appropriate to use... This one of the very few cases...

File: bin/update-apis.js [hunk 2]
```diff
@@ -145,77 +145,42 @@ program
...
-          } else {
-            console.log("Received changes from: " + api.referenceUrl);
-          }
-          api.reference = res.body;
+    p = p.then(function(manifest) {
+      return Promise.all(Object.keys(manifest).map(function(name) {
+        console.log('Fetching %s reference', name); 
+        return request.get(manifest[name]).end().then(function(res) {
+          console.log('Fetched ' + name);
+          apis[name] = {
```
Comment (jonasfj) [inline bin/update-apis.js]: When using manifest like this we should reset `apis = {}` when doing an update. Otherwise we will never remove entries...

**Comment by jonasfj** [comment]
I'll merge and fix the two nits... should be easy :)

**Comment by jonasfj** [comment]
done...

**Comment by jhford** [comment]
:+1:

**Pull request merged**